lib/suggestion-list-element.js

```diff
@@ -417,7 +449,7 @@ module.exports = class SuggestionListElement {
     }
   }
 
-  getDisplayHTML (text, snippet, displayText, replacementPrefix) {
+  getDisplay (text, snippet, displayText, replacementPrefix) {
```

Comment:
Minor nitpick: This name is a bit weird to me. `getElement` maybe?

Reply:
Oh, it's already called "element" in terms of element in the list, so my suggestion isn't good.